src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery[] lotteries = {new Lottery(), new Lottery(), new Lottery()};
+        for (Lottery lottery: lotteries) {
+            System.out.println(lottery.getRandomBall());
+        }
```

Comment:
No need to create 3 different lotteries. Create one lottery and use it to create 3 random balls